src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier rndFigure = new FigureSupplier();

```

Comment:
```suggestion
        FigureSupplier figureSuplier = new FigureSupplier();
```
Don’t  use abbreviations in variable names

Reply:
not fixed